doc/v2/documentation_data_format.rst

```diff
@@ -14,187 +15,375 @@ This document explains the PEtab data format.
+| experimentId        | time              | conditionId     |
++=====================+===================+=================+
+| PETAB_ID            | NUMERIC or '-inf' | CONDITION_ID    |
++---------------------+-------------------+-----------------+
+| timecourse_1        |                 0 | condition_1     |
++---------------------+-------------------+-----------------+
+| timecourse_1        |                10 | condition_2     |
++---------------------+-------------------+-----------------+
+| timecourse_1        |               250 | condition_3     |
++---------------------+-------------------+-----------------+
+| patient_3           |              -inf | condition_1     |
++---------------------+-------------------+-----------------+
+| patient_3           |                 0 | condition_2     |
++---------------------+-------------------+-----------------+
+| intervention_effect |               -20 | no_lockdown     |
++---------------------+-------------------+-----------------+
+| intervention_effect |                20 | mild_lockdown   |
++---------------------+-------------------+-----------------+
+| intervention_effect |                40 | severe_lockdown |
++---------------------+-------------------+-----------------+
+
+The order of the rows is arbitrary, but specifying the rows in ascending order
+of time may improve human readability.
+
+Detailed field description
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The time courses table has three mandatory columns ``experimentId``,
+``time``, and ``conditionId``:
+
+- ``experimentId`` [PETAB_ID, REQUIRED]
+
+  A unique identifier for the experiment, referenced by the ``experimentId``
+  column in the :ref:`v2_measurements_table`.
+
+- ``time``: [NUMERIC or ``-inf``, REQUIRED]
+
+  The time at which the specified condition will become active, in the time
+  unit specified in the model. ``-inf`` indicates that the respective
+  condition will be simulated until a steady state is reached.
+
+  If the simulation of a condition with steady state fails to reach a steady state,
+  and the condition is required for the evaluation of simulation at
+  measurement points, the evaluation of the model is not well-defined.
+  In such cases, PEtab interpreters should notify the user, for example, by
+  returning ``NaN`` or ``inf`` values for the objective function.
+  PEtab does not specify a numerical criterion for steady states.
+  Any event triggers defined in the model must also be checked during this
+  pre-simulation.
+
+  Multiple conditions cannot be applied simultaneously within a single
+  experiment; each ``time`` values must be unique for each ``experimentId``.
+
+  ``time`` will override any initial time specified in the model,
+  except in the case of ``time`` =  `ìnf``, in which case the model-specified
+  time will be used (or 0, if the model does not explicitly specify an initial
+  time).
+
+
+- ``CONDITION_ID``:
+  [PETAB_ID or empty, REQUIRED, REFERENCES(condition.conditionId)]
```

Comment:
```suggestion
- ``conditionId``:
  [CONDITION_ID or empty, REQUIRED, REFERENCES(condition.conditionId)]
```

Reply:
Hm not sure if this is correct. I guess it should be "PETAB_ID", but the table above it says "CONDITION_ID".